Backend/api/validator.py

```diff
@@ -4,8 +4,9 @@
 from django.conf import settings
 from .models import EmailCode
 
-CODE_MIN = 1000000000
-CODE_MAX = 9999999999
+CODE_MIN = 1000
+VERIFICATION_CODE = 8564
```

Comment:
did you make sure the frontend accepts 4 digits now? also I think it is better now with 4 digits

Reply:
Yes, this is how I was able to present to the customer, please check the video